Tasks/DownloadBuildArtifact/typings/globals/node/typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "resolution": "main",
```

Comment:
Ahhh - we want to get away from typings.  Can you do npm install @types with this task?  Please set the right pattern going forward.  We already npm install if package.json exists so I think it should just work.